tests/js/web/NavMainComponentTest.js

```diff
@@ -224,4 +260,102 @@ describe('NavMain component', function() {
 
     });
 
+    describe("when the window is too short for the whole primary navigation to be displayed", function () {
+
+        beforeEach(function() {
+            this.navMainComponent.init();
+        });
+
+        it('should show the more icon link', function () {
+            expect(this.$html.find('.nav-primary .nav-items [label="More"]').is(':visible')).to.be.true;
+        });
+
+
```

Comment:
There's some inconsistent spacing in here too, I'd personally remove the line breaks immediately after the describe lines and at the end to keep them consistent with the beforeEach function block